docs/setup/independent/install-kubeadm.md

```diff
@@ -119,9 +119,16 @@ For each machine:
   The kubelet is now restarting every few seconds, as it waits in a crashloop for
   kubeadm to tell it what to do.
 
-  Note: Disabling SELinux by running `setenforce 0` is required to allow
-  containers to access the host filesystem, which is required by pod networks for
-  example. You have to do this until SELinux support is improved in the kubelet.
+  Note: 
+  - Disabling SELinux by running `setenforce 0` is required to allow containers to access the host filesystem, which is required by pod networks for example. You have to do this until SELinux support is improved in the kubelet.
```

Comment:
Add a newline before the unordered list so that the markdown is rendered correctly. You can preview it here: https://deploy-preview-4340--kubernetes-io-master-staging.netlify.com/docs/setup/independent/install-kubeadm/#installing-kubelet-and-kubeadm